tests/conftest.py

```diff
@@ -2064,58 +2032,28 @@ def multiregion_mirror_setup_fixture(
     ) = multiregion_resources
 
     # Define backing stores
-    backingstore1 = {
-        'name': helpers.create_unique_resource_name(
-            resource_description='testbs',
-            resource_type='s3bucket'
-        ),
-        'region': f'us-west-{randrange(1, 3)}'
-    }
-    backingstore2 = {
-        'name': helpers.create_unique_resource_name(
-            resource_description='testbs',
-            resource_type='s3bucket'
-        ),
-        'region': 'us-east-2'
-    }
-    # Create target buckets for them
-    mcg_obj.create_new_backingstore_aws_bucket(backingstore1)
-    mcg_obj.create_new_backingstore_aws_bucket(backingstore2)
-    aws_buckets.extend((backingstore1['name'], backingstore2['name']))
-    # Create a backing store secret
-    backingstore_secret = mcg_obj.create_aws_backingstore_secret(
-        backingstore1['name'] + 'secret'
-    )
-    backingstore_secrets.append(backingstore_secret)
-    # Create AWS-backed backing stores on NooBaa
-    backingstore_obj_1 = mcg_obj.oc_create_aws_backingstore(
-        backingstore1['name'],
-        backingstore1['name'],
-        backingstore_secret.name,
-        backingstore1['region']
-    )
-    backingstore_obj_2 = mcg_obj.oc_create_aws_backingstore(
-        backingstore2['name'],
-        backingstore2['name'],
-        backingstore_secret.name,
-        backingstore2['region']
+    created_backingstores = backingstore_factory(
+        'OC',
+        {
+            'aws': [(1, 'us-west-1'), (1, 'us-east-2')]
```

Comment:
maybe better to save the regions in constants.py?

Reply:
I'm not sure about its benefits. The vars names wouldn't be shorter or more descriptive, and in case a change will be needed, a global `replace` should take care of all cases automatically